### PR TITLE
Move custom packages from builtin into own repo.

### PR DESCRIPTION
**Pull request by kgerheiser**
This makes it easier to keep track of what packages are in the upstream Spack repository and which packages are custom.

We're already hosting the bundles repo, so I think it's appropriate to have these separate as well. Over time as packages that are able to be merged upstream can be removed.

Also adds CRTM Fix which was not properly transferred with the others.

### CODE REVIEW

**Comment by kgerheiser** [comment]
Just moving some things around for clarity and adding a missing package. Going to merge.

File: var/spack/repos/jcsda-emc/packages/crtm-fix/package.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```
Comment (climbfuji) [inline var/spack/repos/jcsda-emc/packages/crtm-fix/package.py:1]: Just wondering, has this been added to the bundles that need it?

Comment (kgerheiser) [reply var/spack/repos/jcsda-emc/packages/crtm-fix/package.py:1]: No. I really don't know in what context these files are used besides GSI. Should it just be anything that uses CRTM? I think it should be a variant like `depends_on(crtm-fix, when='+fix)`, in whatever bundle it goes in.

Comment (climbfuji) [reply var/spack/repos/jcsda-emc/packages/crtm-fix/package.py:1]: Understood. If the package isn't needed except for GSI, then we can do as you suggest, create a variant and have it off by default.

**Pull request merged**